demisto_sdk/commands/download/downloader.py

```diff
@@ -1,741 +1,978 @@
+                    )
+                    return 1
+
+                content_item_type = self.system_item_type
+                downloaded_content_objects = self.fetch_system_content(
+                    content_item_type=content_item_type
+                )
+
+            else:  # Custom content
+                if not any(
+                    (self.input_files, self.download_all_custom_content, self.regex)
+                ):
+                    logger.error(
+                        "Error: No input parameter has been provided "
+                        "('-i' / '--input', '-r' / '--regex', '-a' / '--all)."
+                    )
+                    return 1
+
+                all_custom_content_data = self.download_custom_content()
+                all_custom_content_objects = self.parse_custom_content_data(
+                    custom_content_data=all_custom_content_data
+                )
+
+                # Filter custom content so that we'll process only downloaded content
+                downloaded_content_objects = self.filter_custom_content(
+                    custom_content_objects=all_custom_content_objects
+                )
+
+                if self.input_files:
+                    downloaded_content_names = [
+                        item["name"] for item in downloaded_content_objects.values()
+                    ]
+
+                    for content_item in self.input_files:
+                        if content_item not in downloaded_content_names:
+                            logger.warning(
+                                f"Custom content item '{content_item}' provided as an input "
+                                f"could not be found / parsed."
+                            )
+                            input_files_missing = True
+
+                if not downloaded_content_objects:
+                    logger.info(
+                        "No custom content matching the provided input filters was found."
+                    )
+                    return 1 if input_files_missing else 0
+
+                if self.auto_replace_uuids:
+                    # Replace UUID IDs with names in filtered content (only content we download)
+                    uuid_mapping = self.create_uuid_to_name_mapping(
+                        custom_content_objects=all_custom_content_objects
+                    )
+
+                    changed_uuids_count = 0
+                    for file_object in downloaded_content_objects.values():
+                        if self.replace_uuid_ids(
+                            custom_content_object=file_object, uuid_mapping=uuid_mapping
+                        ):
+                            changed_uuids_count += 1
+
+                    if changed_uuids_count > 0:
+                        logger.info(
+                            f"Replaced UUID IDs with names in {changed_uuids_count} custom content items."
+                        )
```

Comment:
why is it important to log it? how does logging the count provide any information if i do not know to which content item i updated the uuid?

Reply:
There is also a log for each replaced UUID (in debug level, to not spam the output).
The count gives an indication that the UUID replacement is working properly (not the exact number, but more like if I downloaded 500 content items, and UUIDs were replaced in just 100, I know something might not be working properly).